Open and Close the Webchat Widget Update (#278)
Cognigy/Webchat: docs/webchat-api.md

# Webchat API

> The `initWebchat()` method resolves a `webchat` object as soon as a connection is established, which can be used to access the Webchat's API.

Reviewer: (optional)
`which` --> the connection or the object? If connection, then fine.
If object, then:

```suggestion
The `initWebchat()` method resolves a `webchat` object as soon as a connection is established. You can use the `webchat` object to access the Webchat's API.
```

or

```suggestion
The `initWebchat()` method resolves a `webchat` object as soon as a connection is established. You can use this object to access the Webchat's API.
```

See it in action:  
[![Edit Using the Webchat API](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/using-the-webchat-api-ppl1v?fontsize=14&hidenavigation=1&theme=dark)

## Open and Close the Webchat Widget

To open or close the Webchat widget from your website once it is loaded, use the following methods:

- `webchat.open()` — opens the Webchat widget. Use this method to open the Webchat widget, for example, when a user clicks a custom button on your website. Note that if the Webchat widget is already open, calling this method will close it.
- `webchat.close()` — closes the Webchat widget. Use this method to close the Webchat widget, for example, after a conversation is completed.

> - `webchat.toggle()` — toggles the Webchat widget between open and closed states. For example, if you have a **Chat with us** button on your website, clicking it once opens the Webchat widget and clicking it again closes it.

Reviewer: `it` --> button or website?
```suggestion
- `webchat.toggle()` — toggles the Webchat widget between open and closed states. For example, if your website has a **Chat with us** button, clicking it once opens the Webchat widget and clicking it again closes it.
```

I would assume that the user clicks Chat with us to open, and then another button to close, not the same.

Author: @cl-dmlindoso, please fix your suggestion

> clicking the it

See it in action:  
[![Edit Sending Messages via the Webchat API](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/using-the-webchat-api-hnd6r?fontsize=14&hidenavigation=1&theme=dark)

## Update Webchat Settings

> It is possible to update the Webchat settings at runtime using the `webchat.updateSettings(...)` method.

Reviewer: ```suggestion
To update the Webchat settings at runtime, use the `webchat.updateSettings(...)` method.
```

> It will receive an object as a parameter, which will update all provided settings while leaving the remaining settings untouched.

Reviewer: ```suggestion
This method receives a settings object as a parameter and updates the settings provided in the object while leaving the remaining settings unchanged.
```
